[DOCS] Clarifies the effect of per-field boosting (#63733)
elastic/elasticsearch: docs/reference/query-dsl/multi-match-query.asciidoc

}
--------------------------------------------------

> <1> The `subject` field's score is multipled by three while the `message` field's score is
> unchanged. Note that this does not imply that the `subject` field is three times as important as the
> `message` field due to the differences in term and document statistics across fields.

Reviewer: ```suggestion
<1> The query multiplies the `subject` field's score by three but leaves the
`message` field's score unchanged.
```
